java/runtime/src/main/java/io/ray/runtime/metric/MetricId.java

```diff
@@ -0,0 +1,58 @@
+package io.ray.runtime.metric;
+
+import com.google.common.base.MoreObjects;
+import java.util.Map;
+import java.util.Objects;
+
+public class MetricId {
```

Comment:
What's the metric id used for? Could you add some comments?

Reply:
Ok.